tools/proto-js-plugin/src/main/java/io/spine/js/generate/index/KnownTypesMap.java

```diff
@@ -41,6 +41,9 @@
  *
  * <p>This class generates the map with all the known types written in the form of
  * "{@linkplain io.spine.type.TypeUrl type-url}-to-JS-type".
+ *
+ * <p>Note that services are not included in the known types map as such types are basically
```

Comment:
@dmitrykuzmin I think this is going to be a problem. We have intentionally included services into known types previously, as we wanted to make our code generation working for services as well.

Can we resolve the issue and _not_ modify the behavior of `KnownTypes`?